website/blog/2017-03-24-krishi-iot-solution.md

```diff
@@ -0,0 +1,12 @@
+---
+layout: blog
+title: Vorto in Krishi IoT Solution
+date:   "2017-03-23"
+sequence: 3
+section: blog
+author: Siva Prasad Katru
+githubid: sivakatru/krishi-IoT
+---
+{% include base.html %}
+
+Vorto has been part of a winning IoT solution [Krishi IoT](http://krishi-iot.blogspot.co.id/2017/02/krishi-iot-final-report.html) that has won the 2nd price at the [Eclipse Open IoT Challenge](https://www.eclipse.org/org/press-release/openiotchallenge3_winners.php)
```

Comment:
I would mention both Bosch initiated open source projects: Vorto and Hono, used in the challenge solution. Can you say a few words on what the solution does and where Vorto plays its part ? At least the reader would be excited to read more and follow the link to the entire article.